en/docs/Learn/Tutorials/subscribe-to-an-api.md

```diff
@@ -1,195 +1,201 @@
+    See the following topics for a description of the concepts that you need to know when subscribing to an API:
 
--   [API visibility and subscription availability](_Key_Concepts_)
--   [Applications](_Key_Concepts_)
--   [Application-level throttling](_Key_Concepts_)
--   [Access tokens](_Key_Concepts_)
+    -   [API visibility and subscription availability](_Key_Concepts_)
+    -   [Applications](_Key_Concepts_)
+    -   [Application-level throttling](_Key_Concepts_)
+    -   [Access tokens](_Key_Concepts_)
 
 
-The examples here use the `PhoneVerification` REST API, which is created in the section [Create and Publish an API](_Create_and_Publish_an_API_) .
+The examples here use the `PhoneVerification` REST API, which is created in the section [Create and Publish an API](/Learn/Tutorials/create-and-publish-an-api).
 
-1.  Sign in to the WSO2 API Store ( `https://<hostname>:9443/store` ) and click on an API (e.g., `PhoneVerification 1.0.0` ) to open it.
+1\.  Sign in to the WSO2 API Developer Portal ( `https://<hostname>:<port>/devportal` ) and click on an API (e.g., `PhoneVerification 1.0.0` ) to open it.
 
-        !!! tip
-    In a [multi-tenanted](https://docs.wso2.com/display/AM210/Configuring+Multiple+Tenants) WSO2 API Manager setup, you can access any tenant's store using the URL `http://<hostname>:<port>/store?tenant=<tenant_name>` .
-
-
-2.  Note the subscription options for the REST API.
-    ![]({{base_path}}/assets/attachments/103327837/103327823.png)
+  ![](../../assets/img/Learn/developer-portal-subscriptions.png)
 
-3.  Click the **Applications** menu and click **Add Application** to create a new application.
-    ![]({{base_path}}/assets/attachments/103327837/103327822.png)
+  In the API overview page, You will have two methods to subscribe an API to an application.You can subscribe to current API choosing an existing application or you can follow a wizard to start from the scratch.In the API Manager 3.0.0, We have introduced a __KEY GENERATION WIZARD__ to guide you through the process of creating and configure application, generate application keys and access token, and finally navigate to try out page.
 
-4.  Enter the name as TestApp and select the per token quota as 50PerMin for the application and click **Add** .
-    ![]({{base_path}}/assets/attachments/103327837/103327821.png)
-5.  Click **APIs** and click the PhoneVerification API to view the API's subscription options.
-6.  Select the application that you just created, a tier, and click **Subscribe** .
-    ![]({{base_path}}/assets/attachments/103327837/103327820.png)
-7.  Click the **View Subscriptions** button when prompted.
-    The **Subscriptions** tab opens.
+!!! tip
+    In a [multi-tenanted](https://docs.wso2.com/display/AM210/Configuring+Multiple+Tenants) WSO2 API Manager setup, you can access any tenant's store using the URL `http://<hostname>:<port>/store?tenant=<tenant_name>` .
```

Comment:
Shall we change store to developer portal?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/326/commits/d3856d812d2db0cbb9b24c6374887d904f23ba61